phpstan-baseline.neon

```diff
@@ -3,2230 +3,1690 @@ parameters:
 		-
 			message: "#^Only booleans are allowed in &&, GraphQL\\\\Language\\\\AST\\\\Node\\|null given on the left side\\.$#"
 			count: 1
-			path: src/Error/Error.php
+			path: src\Error\Error.php
```

Comment:
Can you check why it regenerated backslashes? Does not seem correct 🤔Maybe bump phpstan if required.

Reply:
This was a flaw in phpstan innards, which was fixed in 0.12.18. I updated our lock to that version, made the necessary tweak to compose.json, and regenerated the file in 6bc5841. Thanks for catching that!